README.md

```diff
@@ -2,6 +2,18 @@
 
 Run Jest against Lightning web components in a Salesforce DX workspace environment.
 
+## Master vs. Prerelease
+
+If you are testing your components against what is currently deployed on Salesforce production server instances, you should be using the `master` branch. The latest release off of this branch is tagged with the npm dist-tag of `latest`.
+
+For testing against sandbox instances, use the `prerelease` branch and the version tagged as `prerelease`.
+
+### Invalid sourceApiVersion found in sfdx-project.json
+
+If you see this error running tests your Salesforce DX project, it most likely means you are using the incorrect version of this project.
```

Comment:
If you see this error __while__ running tests __on (or maybe in?)__ your Salesforce DX project, it most likely means you are using the incorrect version of this project.